src/main/cpp/Monopoly.cpp

```diff
@@ -118,43 +121,79 @@ Monopoly::getCurrentTile() {
     return board->getTile(loc);
 }
 
+/* Separate this out to two different functions, one for each tile */
+void
+Monopoly::landOnRailroadTile(int multiply) {
```

Comment:
Alot of these "landOn" functions could be written as one template function. Something maybe worth looking into. If I get some time I can write up an example.

Reply:
That sounds good! We could use a switch statement.